File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.py
def solution(order):
    amount  = 0
    coffees = {'americano': 4500 , 'cafelatte': 5000, 'anything': 4500} 

    for coffee in order:
        if 'americano' in coffee:
            amount += coffees['americano']
        elif 'cafelatte' in coffee:
            amount += coffees['cafelatte']
        elif 'anything' in coffee:
            amount += coffees['anything']

    return amount

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        int amount = 0;
        int americano_price = 4500;
        int cafelatte_price = 5000;
        for(String coffee:order) {
            if(coffee.contains("americano") || coffee.contains("anything"))
                amount += americano_price;
            if(coffee.contains("cafelatte"))
                amount += cafelatte_price;
        }
        return amount;
    }
}